internal/scti/otel.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2025 The Tessera authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scti
+
+import (
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"k8s.io/klog/v2"
+)
+
+const name = "github.com/transparency-dev/static-ct/internal/scti"
+
+var (
+	meter  = otel.Meter(name)
+	tracer = otel.Tracer(name)
+)
+
+var (
+	codeKey      = attribute.Key("tesseract.code")
```

Comment:
I think this should be `status_code` from https://opentelemetry.io/docs/specs/semconv/http/http-metrics/